resources/migrations/000_migrations.yaml

```diff
@@ -8340,6 +8340,20 @@ databaseChangeLog:
                                   'DateTimeField', 'DecimalField', 'DictionaryField', 'FloatField', 'IntegerField',
                                   'TextField', 'TimeField', 'UUIDField', 'UnknownField');
 
+  - changeSet:
+      id: 307
+      author: dpsutton
+      comment: 'Added 0.40.0 - Add type to collections'
+      changes:
+        - addColumn:
+            tableName: collection
+            columns:
+              - column:
+                  name: type
```

Comment:
Don't forget to "claim" your migration number in the Slack `#migrations` channel... Tim is already using 307 for BUCM.

Also to avoid the same sort of semantic/relationship/"special" confusion we have with Field types, is it worth naming this column something more specific? I'm not 100% sure what name would be... `canonicality`? I haven't been able to find what the future things beyond `regular` or `official` might go in here.

The only reason I say anything is we also have the Collection `namespace` column to distinguish regular Collections from the special `snippet` "Folders" Collections which have their own Root Collection and hierarchy. I could see myself getting confused and typing `WHERE collection.type = 'snippet'` when I should have typed `WHERE collection.namespace = 'snippet'`. I don't think I'd make that mistake with `WHERE collection.canonicality_type = 'snippet'` or whatever

Reply:
I'm a big fan of not naming this type. The only values that inhabit this at the moment are "official" and nil. It's not clear to me where it might extend to. I believe i've seen "draft" on the table. Hard to know what to name it to but I'm quite open to renaming it if we can think of something better.